src/Xamarin.Android.Build.Tasks/Xamarin.Android.Common.targets

```diff
@@ -680,6 +680,12 @@ because xbuild doesn't support framework reference assemblies.
 		/>
 	</CreateProperty>
 
+	<CreateProperty Value="$(MonoAndroidBinDirectory)">
```

Comment:
Should we start using `<PropertyGroup/>` here? :-)

Reply:
I started a new `<PropertyGroup/>` and we can port all these in the future.